CLI/dsi2lsl.c

```diff
@@ -12,6 +12,7 @@ Copyright (C) 2014-2020 Syntrogi Inc dba Intheon.
 #include <string.h>
```

Comment:
Add some documentation about how this file works and why we added some threading.